Altis_Life.Altis/core/cop/fn_copMarkers.sqf

```diff
@@ -8,34 +8,34 @@
 private _markers = [];
 private _cops = [];
 
-sleep 0.5;
-if (visibleMap) then {
-    {if (side _x isEqualTo west) then {_cops pushBack _x;}} forEach playableUnits; //Fetch list of cops / blufor
+{if (side _x isEqualTo west) then {_cops pushBack _x;}} forEach playableUnits; //Fetch list of cops / blufor
 
-    //Create markers
-    {
-        if !(_x isEqualTo player) then {
-            _marker = createMarkerLocal [format ["%1_marker",_x],visiblePosition _x];
-            _marker setMarkerColorLocal "ColorBLUFOR";
-            _marker setMarkerTypeLocal "Mil_dot";
-            _marker setMarkerTextLocal format ["%1", _x getVariable ["realname",name _x]];
-
-            _markers pushBack [_marker,_x];
-        };
-    } forEach _cops;
+//Create markers
+{
+    if !(_x isEqualTo player) then {
+        _marker = createMarkerLocal [format ["%1_marker",_x],visiblePosition _x];
+        _marker setMarkerColorLocal "ColorBLUFOR";
+        _marker setMarkerTypeLocal "Mil_dot";
+        _marker setMarkerTextLocal format ["%1", _x getVariable ["realname",name _x]];
 
-    while {visibleMap} do {
-        {
-            private _unit = _x select 1;
-            if (!isNil "_unit" && !isNull _unit) then {
-                (_x select 0) setMarkerPosLocal (visiblePosition _unit);
-            };
-        } forEach _markers;
-        if (!visibleMap) exitWith {};
-        sleep 0.02;
+        _markers pushBack [_marker,_x];
     };
+} forEach _cops;
 
-    {deleteMarkerLocal (_x select 0);} forEach _markers;
-    _markers = [];
-    _cops = [];
+while {visibleMap} do {
+    {
+        _x params [
+            "_mark",
+            ["_unit",objNull,objNull]
+        ];
+        if (!isNil "_unit" && !isNull _unit) then {
+            _mark setMarkerPosLocal (visiblePosition _unit);
+        };
+    } forEach _markers;
+    if (!visibleMap) exitWith {};
+    sleep 0.02;
 };
+
+{deleteMarkerLocal (_x select 0);} forEach _markers;
```

Comment:
Could do a new line as well as change select to param, as well as foreach -> count